dub.sdl

```diff
@@ -35,10 +35,7 @@ subPackage {
     "DMDLIB"
 
   preGenerateCommands `
-    "$${DUB_EXE}" \
-    --arch=$${DUB_ARCH} \
-    --compiler=$${DC} \
-    --single "$${DUB_PACKAGE_DIR}config.d" \
+    "$${DUB_PACKAGE_DIR}config.d" \
```

Comment:
For some reason, this dub call doesn't incrementally compile, but calling it directly does.